geoportailv3/static/js/maincontroller.js

```diff
@@ -151,6 +154,19 @@ app.MainController.prototype.manageSelectedLayers_ = function(scope) {
     }
   }
 
+  ngeoSyncArrays(this['map'].getLayers().getArray(),
+      this['selectedLayers'], true, scope,
+      goog.bind(function(layer) {
+        return goog.array.indexOf(
+            this['map'].getLayers().getArray(), layer) !== 0;
+      }, this)
+  );
+  scope.$watchCollection(goog.bind(function() {
+    return this['selectedLayers'];
+  }, this), goog.bind(function() {
+    this['map'].render();
+  }, this));
+
```

Comment:
You need to remove the old code in the `manageSelectedLayers_`. Your code should replace that old code.